test/embulk/input/test_mixpanel.rb

```diff
@@ -1,11 +1,63 @@
 require "prepare_embulk"
+require "embulk/input/mixpanel_api/client"
```

Comment:
This require is needed? (Already required by "embulk/input/mixpanel_api/client")